src/execute.rs

```diff
@@ -426,10 +426,346 @@ impl Execute for System {
+
+        Ok(cycles)
+      }
+
+      0xEB => {
+        // SBC (same as official sbc)
+        let (value, cycles) = self.fetch_operand_value(opcode);
+        self.registers.alu_subtract(value);
+        Ok(cycles)
+      }
+
+      0x9C => {
+        // SHY: (Y & (high(addr) + 1)) -> addr
+        let (address, cycles) = self.fetch_operand_address(opcode);
+        let value = address.to_be_bytes().iter().next().unwrap().clone();
+        let result = self.registers.y & (value.wrapping_add(1));
+        self.registers.sr.set_nz(result);
+        self.write(address, result);
+
+        Ok(cycles + 2)
+      }
+
+      0x9E => {
+        // SHX: (X & (high(addr) + 1)) -> addr
+        let (address, cycles) = self.fetch_operand_address(opcode);
+        let value = address.to_be_bytes().iter().next().unwrap().clone();
+        let result = self.registers.x & (value.wrapping_add(1));
+        self.registers.sr.set_nz(result);
+        self.write(address, result);
+
+        Ok(cycles + 2)
+      }
+
+      0x93 | 0x9F => {
+        // AHX: (A & X & (high(addr) + 1)) -> addr
+        let (address, cycles) = self.fetch_operand_address(opcode);
+        let value = address.to_be_bytes().iter().next().unwrap().clone();
+        let result = self.registers.a & self.registers.x & (value.wrapping_add(1));
+        self.registers.sr.set_nz(result);
+        self.write(address, result);
+
+        Ok(cycles + 2)
+      }
+
+      0x9B => {
+        // TAS: TSX with accumulator and AHX
+        // A AND X -> SP
+        // A AND X AND (H+1) -> M
+        self.registers.sp.set(self.registers.a & self.registers.x);
+
+        let (address, cycles) = self.fetch_operand_address(opcode);
+        let value = address.to_be_bytes().iter().next().unwrap().clone();
+        let result = self.registers.a & self.registers.x & (value.wrapping_add(1));
+        self.write(address, result);
+
+        Ok(cycles + 2)
+      }
+
+      0xBB => {
+        // LAS: LDA + TSX unholy matrimony
+        // M AND SP -> A, X, SP
+        let (value, cycles) = self.fetch_operand_value(opcode);
+        let result = value & self.registers.sp.get();
+
+        self.registers.a = result;
+        self.registers.x = result;
+        self.registers.sp.set(result);
+        self.registers.sr.set_nz(result);
+
+        Ok(cycles)
+      }
+
+      0xAB => {
+        // ATX or LXA: XAA but instead of and X we store in X
+        let (value, cycles) = self.fetch_operand_value(opcode);
+        let magic: u8;
+        #[cfg(not(target_arch = "wasm32"))]
+        {
+          magic = rand::random::<u8>();
+        }
+        #[cfg(target_arch = "wasm32")]
+        {
+          magic = 0xFF;
+        }
+        self.registers.a |= magic;
+        self.registers.a &= value;
+        self.registers.x = self.registers.a;
+        self.registers.sr.set_nz(self.registers.a);
+
+        Ok(cycles)
+      } /* @breqdev change to unreachable or entirely omit?
```

Comment:
@breqdev also hey ;)

Reply:
Let's just remove this entirely, since we map all of the available opcodes!